packages/kit/src/install-fetch.js

```diff
@@ -1,7 +1,20 @@
-// @ts-nocheck
 import fetch, { Response, Request, Headers } from 'node-fetch';
 
-globalThis.fetch = fetch;
-globalThis.Response = Response;
-globalThis.Request = Request;
-globalThis.Headers = Headers;
+Object.defineProperties(globalThis, {
```

Comment:
`defineProperties` seems like a more robust solution, and also avoids having to slap `ts-nocheck` directives.